src/web/app/pages-help/recover-session-links/link-recovery-page.component.ts

```diff
@@ -0,0 +1,93 @@
+import { Component, OnInit, ViewChild } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { ReCaptcha2Component } from 'ngx-captcha';
+import { HttpRequestService } from '../../../services/http-request.service';
+import { StatusMessageService } from '../../../services/status-message.service';
+import { MessageOutput } from '../../../types/api-output';
+import { ErrorMessageOutput } from '../../error-message-output';
+
+/**
+ * Student recover session links page.
+ */
+@Component({
+  selector: 'tm-student-recover-session-links-page',
+  templateUrl: './link-recovery-page.component.html',
+  styleUrls: ['./link-recovery-page.component.scss'],
+})
+export class LinkRecoveryPageComponent implements OnInit {
+
+  // ngx-recaptcha2 element properties
+  captchaSuccess: boolean = false;
+  captchaResponse?: string;
+  size: 'compact' | 'normal' = 'normal';
+  lang: string = 'en';
+
+  formLinkRecovery!: FormGroup;
+  submitted: boolean = false;
+  readonly siteKey: string = '6LeZSZEUAAAAAO-xCzi314NoCgJILEH9qzFuer3P';
```

Comment:
This is a configuration value and cannot be part of the code.

Reply:
Thanks for pointing it out. Moved it to the config template file